app/__init__.py

```diff
@@ -26,12 +32,23 @@
 logging.basicConfig(level=logging.DEBUG)
 
 app = Quart(__name__)
-app = cors(app)
 
 for key in config.keys():
     app.config[key] = config[key]
 
+app = cors(app, allow_headers=['X-Requested-With'], allow_origin="*")
```

Comment:
Allow_origin will be the placeholder for when we will need to limit the gateway access to specific domains. But I was wondering if this would prevent from having custom UI or services using also the APIs? (so I left it open for the moment with "*")